### ISSUE TITLE
About Setting up your computer Page

### ISSUE THREAD

**Issue by rocchow** [econchick/new-coder#201]
I am a newbie but I think I found that there is something wrong on this page.
http://newcoder.io/begin/setup-your-machine/#linux

I am a Ubuntu user. The description about **virtualenv & virtualenvwrapper** for Ubuntu user should be the upper part of code. 

Let me know if I am wrong.

**Comment by merwok** [comment]
I don’t understand your request. Could you rephrase, or post a screenshot with arrows?

**Comment by Hamfri** [comment]
I second him too. The upper part is for Ubuntu and the lower part is for RHEL/Fedora/centos

**Comment by merwok** [comment]
I suspect that the paths changed on Ubuntu, or that the code was written before the deb packaging was changed, or it can be an issue of “installed with pip” vs. “installed with apt”.

The debian package installs things in /usr/share: http://anonscm.debian.org/cgit/collab-maint/virtualenvwrapper.git/tree/debian/virtualenvwrapper.bash-completion

**Comment by cstcyr** [comment]
I can confirm I needed to use "echo 'source /etc/bash_completion.d/virtualenvwrapper' >> ~/.bash_profile" on Linux Mint 17.1 (would have followed the Ubuntu/Debian instructions)
